From 754fc0c720eb998b8e47e695c12807ced0ff3602 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Zbigniew=20J=C4=99drzejewski-Szmek?= <zbyszek@in.waw.pl>
Date: Fri, 3 Oct 2014 08:58:40 -0400
Subject: [PATCH] fileio-label: return error when writing fails

The status of actually writing the file was totally ignored.
---
 src/shared/fileio-label.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/shared/fileio-label.c b/src/shared/fileio-label.c
index c3def3c568..d5ce24c0d9 100644
--- a/src/shared/fileio-label.c
+++ b/src/shared/fileio-label.c
@@ -34,7 +34,7 @@ int write_string_file_atomic_label(const char *fn, const char *line) {
         if (r < 0)
                 return r;
 
-        write_string_file_atomic(fn, line);
+        r = write_string_file_atomic(fn, line);
 
         label_context_clear();
 
@@ -48,7 +48,7 @@ int write_env_file_label(const char *fname, char **l) {
         if (r < 0)
                 return r;
 
-        write_env_file(fname, l);
+        r = write_env_file(fname, l);
 
         label_context_clear();
 
